app/views/curation_concerns/base/_find_work_widget.html.erb

```diff
@@ -0,0 +1,9 @@
+<input class="new-form-control string multi_value optional related_works_ids work_child_members_ids form-control multi-text-field ui-autocomplete-input"
```

Comment:
Is there a reason not to use the rails form helpers here? e.g. `f.input :ordered_member_ids, type: :text...`?